### PR TITLE
Homework1: Roman Snetkov

### PR DESCRIPTION
**Pull request by romajmg**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/Homework1/program10.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# 10 lines: Time, conditionals, from..import, for..else
+
+
+from time import localtime
```
Comment (belovmd) [inline src/Homework1/program10.py]: here you have 1 line between imports and code

File: src/Homework1/program6.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# 6 lines: Import, regular expressions
+
+
+import re
```
Comment (belovmd) [inline src/Homework1/program6.py]: here you have 0 lines between imports and code (earlier - 1). Your code should be consistent

**Review by belovmd — CHANGES_REQUESTED** [review]

File: src/Homework1/program10.py (hunk 1 above)
Comment (romajmg) [reply src/Homework1/program10.py]: Left 1 line beetween import and code in all files

File: src/Homework1/program6.py (hunk 2 above)
Comment (romajmg) [reply src/Homework1/program6.py]: Left 1 line beetween import and code in all files

**Review by belovmd — APPROVED** [review]
1

**Pull request closed without merging**